tox.ini

```diff
@@ -23,6 +23,7 @@ deps =
 	django21: Django>=2.1,<2.2
```

Comment:
you can remove this line since it's unused
```suggestion
```

Reply:
Thanks!